assets/css/components/_kw-pagination.scss

```diff
@@ -0,0 +1,4 @@
+.kw-pagination{
```

Comment:
```suggestion
.kw-pagination {
```

Reply:
We have been using stylelint on the project and it works pretty well.

All we need to do is install it in assets, here is our config in case if you want to try

```
# devDependencies
"stylelint": "^13.12.0",
"stylelint-config-standard": "^21.0.0"
```

Our `.stylelintrc.json`

```json
{
  "extends": [
    "stylelint-config-standard"
  ],
  "rules": {
    "at-rule-no-unknown": [
      true,
      {
        "ignoreAtRules": [
          "extends",
          "apply",
          "tailwind",
          "layer",
          "screen"
        ]
      }
    ]
  }
}
```

In `mix.exs`

```elixir
# For us it would be in `mix codebase`

lint: [
  "format --check-formatted",
  "credo",
  "cmd ./assets/node_modules/.bin/stylelint --color ./assets/css"
],
"lint.fix": [
  "format",
  "cmd ./assets/node_modules/.bin/stylelint --color --fix ./assets/css"
]
```

We also have eslint installed as well. It does help identifying issues.

Reply:
thanks for the suggestion, added. :)